server/render.js

```diff
@@ -153,6 +153,7 @@ export async function serveStaticWithGzip (req, res, path) {
     await serveStatic(req, res, gzipPath)
   } catch (ex) {
     if (ex.code === 'ENOENT') {
+      res.statusCode = 200
```

Comment:
This seems like a legit thing to do.
But, I am fixing the root cause here in #640 
We'll take it pretty soon.